src/controllers/auth.js

```diff
@@ -0,0 +1,115 @@
+import { URL } from 'url'
+import { randomBytes } from 'crypto'
+
+import AppConstants from '../app-constants.js'
+import { getSubscriberByEmail, addSubscriber, updateFxAData } from '../db/index.js'
+// import { sendEmail, getEmailCtaHref, getUnsubscribeUrl } from '../email-utils'
+import { getProfileData, FxAOAuthClient } from '../utils/fxa.js'
+// import { getBreachesForEmail } from '../utils/hibp.js'
+import { fluentError } from '../utils/fluent.js'
+import mozlog from '../utils/log.js'
+const { SERVER_URL } = AppConstants
+
+const log = mozlog('controllers.auth')
+
+export function init (req, res, next, client = FxAOAuthClient) {
```

Comment:
Hmm I don't really mind either way. To me, the good thing about exporting on the function itself is that it's easy to see while scanning the code which function is usable (instead of having to scroll to the bottom).

I can refactor to do a single one at the bottom